dev/NumberBox/APITests/NumberBoxTests.cs

```diff
@@ -49,7 +52,7 @@ public void VerifyTextAlignmentPropogates()
         [TestMethod]
         public void VerifyNumberBoxCornerRadius()
         {
-            if (PlatformConfiguration.IsOSVersionLessThan(OSVersion.Redstone5))
+            if (Common.PlatformConfiguration.IsOSVersionLessThan(Common.OSVersion.Redstone5))
```

Comment:
Why is `Common` needed here?

Reply:
This is because the type PlatformConfiguration exists both in the Common namespace and the MUXControls.TestAppUtils namespace. Adding Common as prefix solves this collision.